code/datums/station_traits/positive_traits.dm

```diff
@@ -98,11 +98,13 @@
 	RegisterSignal(SSdcs, COMSIG_GLOB_JOB_AFTER_SPAWN, PROC_REF(on_job_after_spawn))
 
 
-/datum/station_trait/scarves/proc/on_job_after_spawn(datum/source, datum/job/job, mob/living/spawned, client/player_client)
+/datum/station_trait/wintergear/proc/on_job_after_spawn(datum/source, datum/job/job, mob/living/spawned, client/player_client)
 	SIGNAL_HANDLER
 	var/scarf_type = pick(scarves)
 
 	spawned.equip_to_slot_or_del(new scarf_type(spawned), ITEM_SLOT_NECK, initial = FALSE)
+	if(SSmapping.config?.map_name == "Ice Box Station")  // extra chilly out
```

Comment:
we do not want hardcoded checks for maps which should be easily replaceable. youll have to figure something else out.

Reply:
do you think checking if it has certain words in the name (ice, cool, chill, etc) would work?

Reply:
nope, nothing relating to the name really fixes the issue. heres the 2 ideas i have:
check for the existence of ice storms as a possible weather on the station levels
add a new thing to map jsons thats basically a string of "where im placed", set to "Space" by default, on icebox it can be "Ice Moon", and then you check if the map is on an ice moon. i dont know if thats good but it might be better